src/plugins/auto/src/cumulative_schedule.cpp

```diff
@@ -217,14 +230,17 @@ bool CumuSchedule::schedule_to_worker_infer_request(ov::threading::Task pipeline
         devices = m_context->m_device_priorities;
     }
     lock.unlock();
-    for (auto&& device : devices) {
-        if (!preferred_device.empty() && (device.device_name != preferred_device)) {
```

Comment:
we have to keep preferred_device code path

Reply:
Updated.